factories_ZA/basePlotter.py

```diff
@@ -551,7 +562,100 @@ def get_jet_flavour_cut(flav, jet_idx):
                         'binning': '(50, 0, 1)'
                 }
             ])
-            
+
+
+            #PLOTS IN ELLIPSE
+            if not self.btag or cat == "MuEl":
+                continue
+            if cat == "MuMu":
+                with open(self.filename_MuMu) as f:
+                    parameters = json.load(f)
+            elif cat == "ElEl":
+                with open(self.filename_ElEl) as f:
+                    parameters = json.load(f)
+            for j, line in enumerate(parameters):
+                if cat == "MuMu":
+                    inWindowCut = "window_MuMu.isInWindow({0}, {1}, 2., {2}, {3})".format(float(line[0]), float(line[1]), self.jj_str + ".M()", self.baseObject+".p4.M()")
+                    self.ellCut = self.joinCuts(self.cutWithoutStagesAndCat, self.dict_cat_cut[cat], self.dict_stage_cut["mll_and_met_cut"], inWindowCut)
+                elif cat == "ElEl":
+                    inWindowCut = "window_ElEl.isInWindow({0}, {1}, 2., {2}, {3})".format(float(line[0]), float(line[1]), self.jj_str + ".M()", self.baseObject+".p4.M()")
+                    self.ellCut = self.joinCuts(self.cutWithoutStagesAndCat, self.dict_cat_cut[cat], self.dict_stage_cut["mll_and_met_cut"], inWindowCut)
+                self.extraString = "inEllipse_{0}_{1}_{2}".format(j, round(line[0], 1), round(line[1], 1)) #Labelling each of the 21 ellipses with its index. The histograms will be named with this label and the reco mbb and mllbb. Will need to write down which ellipse corresponds to which index.
+                self.extraStringForInOut = "{0}_{1}_{2}".format(j, round(line[0], 1), round(line[1], 1))
```

Comment:
if this is about the index why `round` instead of `int`, any particular reason ?
(same comment for out of ellipse)